### ISSUE TITLE
Problem with yaml schema updating

### ISSUE THREAD

**Issue by boggeroff** [redhat-developer/vscode-yaml#639]
## Describe the bug

When modifying and saving a .yaml schema file, changes are not reflected where it is used only after reloading window.

## Expected Behavior

With .json schema files, the changes are reflected instantly after saving the file.

## Current Behavior

Some kind of hard restart is needed to see the changes.

## Steps to Reproduce

1. Have a yaml schema
2. Have a yaml data file
3. Have a yaml.schemas mapping between the schema and the data file
4. Update the schema and see for yourself

## Environment
- [ ] Windows
- [ ] Mac
- [x] Linux
- [ ] other (please specify)

**Comment by corneliusroemer** [comment]
I also encountered this bug when setting up a schema for other ymls in a workspace.

In my case this is on macOS 12.5 (M1 = apple silicon = ARM)

Workaround for me: Reload workspace using `cmd + shift + P; then type reload; hit enter`

**Comment by corneliusroemer** [comment]
Duplicate of #309 

Would be great if someone from red-hat could have a look at this - it's kind of a major issue when setting up your own schema and trying to use it for validation. @msivasubramaniaan

There are in total around 10 likes on this bug (and its duplicate(s))
